lib/benchee/statistics.ex

```diff
@@ -159,7 +160,7 @@ defmodule Benchee.Statistics do
     deviation           = standard_deviation(run_times, average, iterations)
     standard_dev_ratio  = deviation / average
     standard_dev_ips    = ips * standard_dev_ratio
-    median              = compute_median(run_times, iterations)
+    median              = Percentile.percentile(run_times, iterations, 50)
```

Comment:
Good call re-using this! I never would have put that together myself.

Reply:
Hahaha, I think you would have. I didn't realize until I kept seeing the same values for median and 50th percentile. And reading!

Reply:
it's non obvious yeah but this is how I imagined it. Thanks :grin: